maintenance_scripts/cleanup.py

```diff
@@ -73,11 +73,16 @@ def cleanup_temp_dir(days_old: int = 7, dry_run: bool = True):
 
 
 def main():
+    print("Updating Spyglass versions table...")
     SpyglassVersions().fetch_from_pypi()
+    print("Running table cleanups...")
     run_table_cleanups()
+    print("Cleaning up external files...")
     cleanup_external_files()
+    print("Cleaning up temporary directory...")
     cleanup_temp_dir(dry_run=False)
 
 
 if __name__ == "__main__":
-    main()
+    pass
```

Comment:
I'm confused by this. You don't want to run this file as a script?

Reply:
Whoops - nope, that was a dev edit I've reverted